plugins/builds/update.js

```diff
@@ -204,11 +205,10 @@ module.exports = () => ({
                 build.stats = Object.assign(build.stats, {
                     blockedStartTime: new Date().toISOString()
                 });
-            } else if (desiredStatus === 'QUEUED') {
+            } else if (desiredStatus === 'QUEUED' && currentStatus !== 'QUEUED') {
```

Comment:
should we skip `build.update()` on line 225 if `desiredStatus` and `currentStatus` are `'QUEUED'`?

Reply:
Maybe my understanding is that it could be updated by the value of `build.stats` passed from [queue-service](https://github.com/screwdriver-cd/queue-service/pull/31/files#diff-ea1f58226f3c81f477d5ed24aacecc91344e9c00c9a9540e080b39e73e06ddcdR399) in line 192.

Reply:
ah okay, so we still need to update